generate/templates/manual/libgit2/opts.cc

```diff
@@ -0,0 +1,139 @@
+NAN_METHOD(GitLibgit2::Opts)
+{
+  Nan::EscapableHandleScope scope;
+
+  if (info.Length() == 0 || !info[0]->IsNumber())
+  {
+    return Nan::ThrowError("Number option is required.");
+  }
+
+  int from_option = (int)info[0].As<v8::Number>()->Value();
+
+  git_error_clear();
+
+  v8::Local<v8::Value> to;
+  switch (from_option)
+  {
+  // GET size_t
+  case GIT_OPT_GET_MWINDOW_SIZE:
```

Comment:
We should indent. Super gross old coding style :).

Reply:
VSCode defaults strike again.